python/plugins/db_manager/db_plugins/spatialite/connector.py

```diff
@@ -571,18 +571,33 @@ def runVacuum(self):
         c.execute('VACUUM')
         self.connection.isolation_level = '' # reset to default isolation
 
+    def commentTable(self, schema, tablename, comment):
+        """Comment the table"""
+        return ''
+
+    def commentTable(self, schema, tablename, comment):
+        return ''
+
+    def unCommentTable(self, schema, tablename):
+        return ''
+
+    def getComment(self, tab, field, db):
+        """Returns the comment for a field"""
+        return ''
+
     def addTableColumn(self, table, field_def):
         """ add a column to table """
         sql = u"ALTER TABLE %s ADD %s" % (self.quoteId(table), field_def)
+        self._execute_and_commit(sql)
         self._execute(None, sql)
 
-        sql = u"SELECT InvalidateLayerStatistics(%s)" % (self.quoteId(table))
-        self._execute(None, sql)
+        # sql = u"SELECT InvalidateLayerStatistics(%s)" % (self.quoteId(table))
```

Comment:
What's this for?

Reply:
I'm not the one writing it. It was dropped from the master because of a bad merge, so i just wrote it again

Reply:
If it shows a difference in this pull request, it means that there's a difference between your branch and current master. Make sure they don't show up like this in here.

Reply:
Aren't the difference related to the last commit of the pull request ?